### PR TITLE
Add/correct translation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: core/src/main/resources/lib/hudson/project/console-link_de.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+View\ as\ plain\ text=Als unformatierten Text anzeigen
```
Comment (daniel-beck) [inline core/src/main/resources/lib/hudson/project/console-link_de.properties]: Isn't this too long for the sidepanel?

**Comment by daniel-beck** [comment]
@albers WDYT?

File: core/src/main/resources/lib/hudson/project/console-link_de.properties (hunk 1 above)
Comment (albers) [inline core/src/main/resources/lib/hudson/project/console-link_de.properties]: Works for me (Firefox on Windows 7):
als-unformatierten-text-anzeigen
LGTM, thanks @phoenix384

File: core/src/main/resources/lib/hudson/project/console-link_de.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# The MIT License
+#
+# Copyright (c) 2016
```
Comment (daniel-beck) [inline core/src/main/resources/lib/hudson/project/console-link_de.properties]: This line makes no sense AFAIU.

**Comment by daniel-beck** [comment]
Thanks @phoenix384 !

**Pull request merged**